src/libexec/zmrestoreblob

```diff
@@ -14,21 +14,80 @@
 # If not, see <https://www.gnu.org/licenses/>.
 # ***** END LICENSE BLOCK *****
 
-#Usage
-# su - zimbra
-# zmrestoreblob -m 3
-# where -m is to specify which mailbox to check. In above case it is mailbox id 3
+=head1 NAME
+zmrestoreblob - restores missing blobs for orphan headers
+
+=head1 SYNOPSIS
+
+  zmrestoreblob [options]
+  Options:
+    -a|account     Check orphan headers for given email account, if founds any missing blobs, restore it
+    -h|help        Display this help message.
+
+  Example:  zmrestoreblob -a test1@zcs-dev.test
+
+=head1 DESCRIPTION
+This utility is used to check orphan headers for given account, if founds any missing blobs, restore it
+
+=cut
 
 use strict;
 use warnings;    #or else.
 use lib "/opt/zimbra/common/lib/perl5";
 use Data::Dumper;
 use File::Basename;
 use File::Copy;
+use Archive::Zip qw( :ERROR_CODES :CONSTANTS );
+use Getopt::Long;
+use Pod::Usage qw(pod2usage);
+
+my $opt_help;
+my $opt_account;
+
+GetOptions(
+    "help|h"      => \$opt_help,
+    "account|a=s" => \$opt_account
+);
```

Comment:
- should also check for invalid options by checking the return value from GetOptions (see ```$ perldoc Getopt::Long```).

Reply:
Done suggested changes and pushed, thanks